src/main.ts

```diff
@@ -304,6 +306,16 @@ async function main() {
 		}
 	});
 
+	// Listen for deleted messages
+	client.on('messageDelete', async message => {
```

Comment:
This event already exists a few lines down

Reply:
this is a frikk'n git merge error